Resources/views/standard_layout.html.twig

```diff
@@ -315,5 +278,47 @@ file that was distributed with this source code.
             </div>
         {% endblock sonata_wrapper %}
     </div>
+
+    {% block javascripts %}
```

Comment:
We cannot move this code at this place. There are some inline js requiring definition on top. Also, for a backing this is not important to respect this convention.